js/components/Header.js

```diff
@@ -81,11 +83,13 @@ Header.propTypes = {
   navigator: PropTypes.object,
 };
 
-Header.HEIGHT = 54;
+const STATUS_BAR_HEIGHT = Platform.OS === 'ios' ? 20 : StatusBar.currentHeight;
```

Comment:
20 is the height of the status bar on iOS. In Android, we can just use `StatusBar.currentHeight`.
54 is the height we want the header to have. This should be pretty self explanatory already.